components/grade-table.js

```diff
@@ -0,0 +1,18 @@
+class GradeTable {
+  constructor (tableElement) {
+    this.tableElement = tableElement;
+  }
+  updateGrades (grades) {
+    var tBody = this.tableElement.querySelector("tbody");
+    for (var i = 0; i < grades.length; i++) {
+      var row = document.createElement("tr");
+      var fullName = document.createElement("td");
+      var course = document.createElement("td");
+      var grade = document.createElement("td");
+      fullName.textContent = grades[i].name;
+      course.textContent = grades[i].course;
+      grade.textContent = grades[i].grade;
+
+      $("tbody").append(row, fullName, course, grade);
```

Comment:
Here everything is done in "core" javascript except line #16. You usually don't want to mix core and jQuery you should stick to one or the other. Also you already have a pointer to the `tbody` on line 6 so on line 16 you should't re-query the DOM with jQuery to get another reference to the tbody element. You should change line 16 to use core JavaScript and utilize the `tbody` var you already created above. Look into [appendChild](https://developer.mozilla.org/en-US/docs/Web/API/Node/appendChild) or [append](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/append)